managed/services/telemetry/config.default.yml

```diff
@@ -936,3 +936,38 @@ telemetry:
     data:
       - metric_name: "postgresql_db_count"
         value: 1
+
+  - id: PMMServerHA
+    source: ENV_VARS
+    summary: "Use of HA feature"
+    transform:
+      type: StripValues
+    data:
+      - metric_name: "pmm_server_ha_ch_host_ip"
+        column: "CH_HOST_IP"
+      - metric_name: "pmm_server_ha_vm_host_ip"
+        column: "VM_HOST_IP"
+      - metric_name: "pmm_server_ha_pg_host_ip"
+        column: "PG_HOST_IP"
+      - metric_name: "pmm_server_ha_pg_username"
+        column: "PG_USERNAME"
+      - metric_name: "pmm_server_ha_pg_password"
+        column: "PG_PASSWORD"
+      - metric_name: "pmm_server_ha_gf_username"
+        column: "GF_USERNAME"
+      - metric_name: "pmm_server_ha_gf_password"
+        column: "GF_PASSWORD"
+      - metric_name: "pmm_server_ha_pmm_active_ip"
+        column: "PMM_ACTIVE_IP"
+      - metric_name: "pmm_server_ha_pmm_active_node_ip"
+        column: "PMM_ACTIVE_NODE_ID"
+      - metric_name: "pmm_server_ha_pmm_passive_ip"
+        column: "PMM_PASSIVE_IP"
+      - metric_name: "pmm_server_ha_pmm_passive_node_id"
+        column: "PMM_PASSIVE_NODE_ID"
+      - metric_name: "pmm_server_ha_pmm_passive2_ip"
+        column: "PMM_PASSIVE2_IP"
+      - metric_name: "pmm_server_ha_pmm_passive2_node_id"
+        column: "PMM_PASSIVE2_NODE_ID"
+      - metric_name: "pmm_server_ha_docker_image"
+        column: "DOCKER_IMAGE"
```

Comment:
env vars should be taken from this block https://www.notion.so/percona/Setup-PMM-in-HA-mode-DRAFT-11fd2dca015d4f288a554ce02b7ac882?pvs=4#0ea56a2fd9044dfb80a77d7d6bc18e7f

Reply:
I was reflecting on whether we really need all of the variables listed there? It seems like all we need to know is whether or not `PERCONA_TEST_HA_ENABLE` is set to 1. The rest of the envvars is not relevant for our stats.

Well, maybe we need a couple more, but certainly not the IPs. :)

What do you think?

Anyway, I updated the list to reflect the vars that make sense from my standpoint. Please review.